plugins/vue-directives.js

```diff
@@ -0,0 +1,36 @@
+import Vue from 'vue';
+
+Vue.directive('visible-on-scroll', {
+  inserted: (el) => {
+    el.scrolledVisible = true;
+    window.addEventListener('scroll', () => handleScroll(el));
+    window.addEventListener('orientationchange', () => handleOrientationChange(el));
+  }
+});
+
+const handleScroll = (el) => {
+  const newPosition = window.scrollY;
+  if (onDesktop && el.scrollPosition < newPosition && el.scrollPosition > 150 && el.scrolledVisible) {
```

Comment:
150 is when the scroll kicks in? If so, wonder if we should pass this value from the vue component?

Reply:
Potentially, but to limit the scope let's leave it hardcoded for now until we have an actual usecase.